chemprop/models/model.py

```diff
@@ -2,13 +2,13 @@
 
 from typing import Iterable
 
-from lightning import pytorch as pl
 import torch
-from torch import nn, Tensor, optim
+from lightning import pytorch as pl
+from torch import Tensor, nn, optim
```

Comment:
Is there a configuration in `isort` that can treat both regular and `from`-style imports the same during sorting? That is, `isort` now separates the lines: `import torch` and `from torch import ...` into different subsections within this grouping. IMO it looks cleaner to keep them grouped together.